nessie/merged/sis_profile.py

```diff
@@ -297,6 +298,12 @@ def merge_sis_profile_plans(academic_status, sis_profile):
             # A plan with no status is considered discontinued. (NS-689)
             plan_feed['status'] = 'Discontinued'
 
+        # Add any subplans.
+        if student_plan.get('academicSubPlan', None):
+            plan_feed['subplan'] = student_plan['academicSubPlan']
```

Comment:
@lyttam - We want `subplan`, not `subPlan`?

Reply:
Camel-cased `academicSubPlan` is a Hub-style feed; lowercase `subplan` is our own feed we're serving up for BOA consumption, so we get to set the conventions.